spec/factories/faq_feedback_factory.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
-  factory :green_lanes_faq_feedback, class: 'GreenLanes::FaqFeedback' do
+  factory :faq_feedback, class: 'FaqFeedback' do
```

Comment:
We actually don't need the class anymore
```suggestion
  factory :faq_feedback do
```